server/controllers/followersController.js

```diff
@@ -0,0 +1,159 @@
+          message: 'You\'ve unfollow this User'
+        });
+    } catch(err) {
+      return next(err);
+    }
+  }
+
+  /**
+   * @description - This function get all user followers using the id supplied
+   * @param {object} req - request to be sent
+   * @param {object} res - response gotten from server
+   * @param {object} next - callback function
+   * @param {string} id - ID
+   * @returns {object} - get user
+   */
+  static async getFollowers(req, res, next, id) {
+    try {
+      const userId = id;
+      const followers = await Follower.findAll({
+        where: {userId: userId}
+      });
+      return followers.length >= 1 ? res.status(200).json({
+        success: true,
+        message: 'Followers returned successfully',
+        followers
+      }):res.status(200).json({
+        success: true,
+        message: 'No followers found'
+      });
+    } catch(err) {
+      return next(err);
+    }
+  }
+
+   /**
+   * @description - This function get user followees using the id supplied
+   * @param {object} req - request to be sent
+   * @param {object} res - response gotten from server
+   * @param {object} next - callback function
+   * @param {string} id - ID
+   * @returns {object} - get user
+   */
+  static async getFollowees(req, res, next, id) {
+    try {
+      const followees = await Follower.findAll({
+        where: {followerId: id}
+      });
+      return followees.length >= 1 ? res.status(200).json({
+        success: true,
+        message: 'Followees returned successfully',
+        followees
+      }):res.status(200).json({
+        success: true,
+        message: 'No followers found'
+      });
+    } catch(err) {
+      return next(err);
+    }
+  }
+  /**
+   * @description - This function get an authenticated user
+   * followers using the userId
+   * @param {object} req - request to be sent
+   * @param {object} res - response gotten from server
+   * @param {object} next - callback function
+   * @returns {object} - Response to be sent
+   */
+  static getAllFollowers(req, res, next) {
+    const userId = req.token.user.id;
+    return FollowerController.getFollowers(req, res, next, userId);
+  }
+
+  /**
+   * @description - This function get an authenticated user
+   * followee using the userId
+   * @param {object} req - request to be sent
+   * @param {object} res - response gotten from server
+   * @param {object} next - callback function
+   * @returns {object} - Response to be sent
+   */
+  static async getAllFollowees(req, res, next) {
+    const id = req.token.user.id;
+    return FollowerController.getFollowees(req, res, next, id);
+  }
+
+   /**
+   * @description - This function get other user followees using the user's ID
+   * @param {object} req - request to be sent
+   * @param {object} res - response gotten from server
+   * @param {object} next - callback function
+   * @returns {object} - Response to be sent
+   */
+  static async getAnotherUserFollowees(req, res, next) {
```

Comment:
@Danielshow is this function really necessary? You're still returning `FollowerController.getFollowers`